Reorganized sidebar for Users section + mass documentation commit

Xyrrm opened the pull request:
Reorganized the sidebar for the Users section, prepping structure for bigger wiki contributions.
Added Ponder documentation from Create Fandom wiki

IThundxr commented on .vitepress/sidebars/users.ts:
```diff
@@ -33,46 +35,71 @@ export default {
+        //  Group addons/integrations together for clearer organization
         {
-          text: "Create's Computercraft Integration",
+          text: "Addons & Integrations",
```
The wiki shouldn't really contain information about addons, there's far too many of them and this wiki is more focused on create itself, if a addon needs a wiki they can host vitepress on gh pages themselves

Xyrrm replied:
gotcha, is CC the only integration?

cassiancc replied:
There's automatic portal integration for The Aether and Better End.

IThundxr reviewed (approved):
Looks good, there will need to be some redirects setup to not break old changelog links etc but i'll handle that myself

Xyrrm commented:
- Added Ponder categories
- Added assets (block & items)
- Added initial batch of documentation pages
- Synced changes from 42065ffb9e54195382c07c6b9c9e47bac12cca5e

Xyrrm commented:
Closed to create separate branch for mass commit
